app/src/main/java/org/greenstand/android/TreeTracker/activities/TreeTrackerActivity.kt

```diff
@@ -1,34 +1,84 @@
 package org.greenstand.android.TreeTracker.activities
 
 import android.os.Bundle
-import android.os.PersistableBundle
-import android.util.Log
-import androidx.appcompat.app.AppCompatActivity
-import androidx.navigation.findNavController
-import org.greenstand.android.TreeTracker.R
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.compositionLocalOf
+import androidx.compose.runtime.getValue
+import androidx.navigation.compose.*
+import org.greenstand.android.TreeTracker.dashboard.DashboardScreen
 import org.greenstand.android.TreeTracker.databinding.TreeTrackerActivityBinding
-import org.greenstand.android.TreeTracker.models.FeatureFlags
-import org.greenstand.android.TreeTracker.models.Language
-import org.greenstand.android.TreeTracker.models.LanguageSwitcher
+import org.greenstand.android.TreeTracker.languagepicker.LanguageSelectScreen
+import org.greenstand.android.TreeTracker.models.*
+import org.greenstand.android.TreeTracker.signup.SignupScreen
+import org.greenstand.android.TreeTracker.splash.SplashScreen
+import org.greenstand.android.TreeTracker.view.TreeTrackerTheme
 import org.koin.android.ext.android.inject
+import timber.log.Timber
 
-class TreeTrackerActivity : AppCompatActivity() {
+val LocalViewModelFactory = compositionLocalOf<TreeTrackerViewModelFactory> { error("No active ViewModel factory found!") }
```

Comment:
This is what I mentioned in the write up. Essentially, this allow the `ViewModelFactory` to exist in the compose runtime and be accessible to all of our views going forward